src/print.jl

```diff
@@ -110,36 +110,33 @@ end
 #------------------------------------------------------------------------
 ## VariableRef
 #------------------------------------------------------------------------
-Base.show(io::IO, v::VariableRef) = print(io, var_str(REPLMode,v))
-Base.show(io::IO, ::MIME"text/latex", v::VariableRef) =
+Base.show(io::IO, v::AbstractVariableRef) = print(io, var_str(REPLMode,v))
```

Comment:
Why do we want extensions to implement `var_str` instead of `Base.show` for their variable type?

Reply:
They only need to implement `JuMP.name`

Reply:
Document please